cloudflare/src/framework/async_api.rs

```diff
@@ -21,6 +24,16 @@ pub trait ApiClient {
         ResultType: ApiResult,
         QueryType: Serialize,
         BodyType: Serialize;
+
+    /// Send a request to a particular Cloudflare API endpoint, get the response as bytes.
+    async fn request_raw_bytes<ResultType, QueryType, BodyType>(
```

Comment:
What's the point of passing ResultType if you're gonna return Bytes?

Reply:
We're not passing ResultType, we're acknowledging that this function can take an endpoint with any ResultType. After all, every ResultType can be expressed as bytes.